src/main/java/com/iland/coda/footprint/CodaClient.java

```diff
@@ -254,6 +310,54 @@ void triggerScan(List<String> targets, List<Integer> scanners,
 	 */
 	ScanStatus getScanStatus(Integer accountId) throws ApiException;
 
+	/**
+	 * Retrieve the collated {@link ScanSurfaceEntry scan surface entries} for all scanners for the given {@link Integer accountId}.
+	 *
+	 * @param accountId Account ID you want to receive request for. If not provided, falls back on <code>original_account_id</code> from the auth endpoint.
+	 * @return a collated {@link ScanSurfaceEntry scan surface entries} for all scanners for the given {@link Integer accountId}
+	 * @throws ApiException
+	 */
+	default Set<ScanSurfaceEntry> getScanSurface(final Integer accountId)
+		throws ApiException {
+		try {
+			return getScannerIdByLabel(accountId).values().stream()
+				.map(scannerId -> {
+					try {
+						return getScanSurface(scannerId, accountId);
+					} catch (ApiException e) {
+						throw new RuntimeException(e);
+					}
+				}).flatMap(Collection::stream).collect(Collectors.toSet());
+		} catch (RuntimeException e) {
+			Throwables.throwIfInstanceOf(e.getCause(), ApiException.class);
+			throw e;
+		}
+	}
+
+	/**
```

Comment:
Should be more like...
Retrieve the collated {@link ScanSurfaceEntry scan surface entries} for the given scanners and the given {@link Integer accountId}.